setup.sh

```diff
@@ -116,16 +119,15 @@ mkdir -p /opt/ioi/store/screenshots
 mkdir -p /opt/ioi/store/submissions
 mkdir -p /opt/ioi/config/ssh
 
-# Latest as of 2023-05-20
-aria2c -x 4 -d /tmp -o cpptools-linux.vsix "https://github.com/microsoft/vscode-cpptools/releases/download/v1.15.4/cpptools-linux.vsix"
-wget -O /tmp/vscodevim.vsix "https://github.com/VSCodeVim/Vim/releases/download/v1.25.2/vim-1.25.2.vsix"
-rm -rf /tmp/vscode
-mkdir /tmp/vscode
-mkdir /tmp/vscode-extensions
-code --install-extension /tmp/cpptools-linux.vsix --extensions-dir /tmp/vscode-extensions --user-data-dir /tmp/vscode
+# Latest as of 2023-07-23
+$wget -O $cache/cpptools-1.16.3.vsix https://github.com/microsoft/vscode-cpptools/releases/download/v1.16.3/cpptools-linux.vsix
+mkdir /tmp/vscode /tmp/vscode-extensions
+code --install-extension $cache/cpptools-1.16.3.vsix --extensions-dir /tmp/vscode-extensions --user-data-dir /tmp/vscode
 tar jcf /opt/ioi/misc/vscode-extensions.tar.bz2 -C /tmp/vscode-extensions .
-cp /tmp/vscodevim.vsix /opt/ioi/misc
-rm -rf /tmp/vscode-extensions
+rm -rf /tmp/vscode /tmp/vscode-extensions
```

Comment:
I think we do not want to delete directories in the setup script. It seemed that the idea all along was that if anything goes bad in the setup, we need to investigate it and understand where it went wrong. Hence there are very few deletions whatsoever.

Adding `/tmp/vscode` is an unrelated change in the PR.